doc/source/conf.py

```diff
@@ -27,7 +27,7 @@
     "additional_breadcrumbs": [
         ("PyAnsys", "https://docs.pyansys.com/"),
         ("PyMAPDL", "https://mapdl.docs.pyansys.com/"),
-        ("Examples", "https://mapdl.docs.pyansys.com/examples/"),
+        ("Examples", "https://mapdl.docs.pyansys.com/release/0.64/examples/index.html"),
```

Comment:
Let's point to ``dev`` better since it is more stable
```suggestion
        ("Examples", "https://mapdl.docs.pyansys.com/dev/examples/index.html"),
```